metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -645,54 +677,87 @@ def build_computed_metrics_node(
             metric_specs=[metric_spec],
         )
 
-    def _measures_for_metric(
+    def _build_metric_input_measure_for_base_metric(
```

Comment:
Note that we're about to add conversion metrics, which will have multiple input metrics - so this might need to be rewritten by @WilliamDee

Reply:
+1 I need this, so we should sync to see whats the best path forward

Reply:
we mean multiple input measures, not metrics, but yes.

Reply:
im not gonna lie, i always get so confused visually and acoustically. Having to say/see metric input measure instead of just input measure messes with my eyes and ears thinking we're talking about input metrics

Reply:
Updated to `input_measure`.